src/pages/Validator/index.tsx

```diff
@@ -4,17 +4,31 @@ import {useParams} from "react-router-dom";
 import PageHeader from "../layout/PageHeader";
 import ValidatorTitle from "./Title";
 import ValidatorDetailCard from "./DetailCard";
+import ValidatorStakingBar from "./StakingBar";
+import {useGetMainnetValidators} from "../../api/hooks/useGetValidatorSet";
+import EmptyValue from "../../components/IndividualPageContent/ContentValue/EmptyValue";
 
 export default function ValidatorPage() {
   const address = useParams().address ?? "";
+  // make sure that addresses will always start with "0x"
+  const addressHex = address.startsWith("0x") ? address : "0x" + address;
+  const {validators} = useGetMainnetValidators();
```

Comment:
can `validators` be null/undefined on first load? do we need to use `useEffect` here?

Reply:
good point! but instead of using useEffect, we can use skeleton to render the loading page. will add it in the following pr!